docs/source/configuration/configuration-gconstruction.rst

```diff
@@ -12,8 +12,8 @@ Graph Construction
 * **-\-num-processes-for-edges**: the number of processes to process edge data simulteneously. Increase this number can speed up edge data processing.
 * **-\-output-dir**: (**Required**) the path of the output data files.
 * **-\-graph-name**: (**Required**) the name assigned for the graph.
-* **-\-remap-node-id**: boolean value to decide whether to rename node IDs or not. Default is true.
-* **-\-add-reverse-edges**: boolean value to decide whether to add reverse edges for the given graph. Default is true.
+* **-\-remap-node-id**: boolean value to decide whether to rename node IDs or not. Adding this argument will set it to be true.
```

Comment:
```suggestion
* **-\-remap-node-id**: boolean value to decide whether to rename node IDs or not. Adding this argument will set it to be true, otherwise false.
```